### PR TITLE
Created CSharp Checkout example

### PR DESCRIPTION
**Pull request by StephenJosey**

### CODE REVIEW

File: connect-examples/v2/csharp_checkout/csharp_checkout/Pages/Checkout.cshtml.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    {
+      CheckoutApi checkoutApi = new CheckoutApi();
+      int amount = (int)float.Parse(Request.Form["amount"]) * 100;
+
+      try
+      {
+
+        CreateCheckoutRequest body = new CreateCheckoutRequest(
+          IdempotencyKey: Guid.NewGuid().ToString(),
+          Order: new CreateOrderRequest(
```
Comment (hukid) [inline connect-examples/v2/csharp_checkout/csharp_checkout/Pages/Checkout.cshtml.cs]: Is this same as other checkout examples?

Can we call `CreateOrderRequest` to create a order before call `CreateCheckoutRequest `?

File: connect-examples/v2/csharp_checkout/csharp_checkout/Pages/Checkout.cshtml.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+              }
+        ));
+
+        CreateCheckoutResponse response = checkoutApi.CreateCheckout(locationId, body);
+        return Redirect(response.Checkout.CheckoutPageUrl);
+      }
+      catch (Square.Connect.Client.ApiException e)
+      {
+        this.ResultMessage = e.Message;
+        return null;
```
[the remark this answers is not in the document]
Comment (StephenJosey) [reply connect-examples/v2/csharp_checkout/csharp_checkout/Pages/Checkout.cshtml.cs]: I'm not sure. What is the other alternative to continue to the page (not redirect)? I'm not very good with C# :P

[the remark this answers is not in the document]
Comment (hukid) [reply connect-examples/v2/csharp_checkout/csharp_checkout/Pages/Checkout.cshtml.cs]: usually, you don't return a page from a POST request. Instead, return a redirect to error page, and use that error page to display error information.

File: connect-examples/v2/csharp_checkout/csharp_checkout/Pages/Checkout.cshtml.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+using Square.Connect.Api;
+using Square.Connect.Model;
+
+namespace csharp_checkout.Pages
+{
+  public class CheckoutModel : PageModel
+  {
+    private readonly string locationId;
+
+    public CheckoutModel(IConfiguration configuration)
```
Comment (hukid) [inline connect-examples/v2/csharp_checkout/csharp_checkout/Pages/Checkout.cshtml.cs]: in mvc, can you create a controller instead of a page?

**Pull request merged**